static/css/main.css

```diff
@@ -189,7 +189,7 @@ body > footer a img {
 
 /* Main */
 main {
-  margin: 10rem 0;
+  margin: 10rem 0 0;
```

Comment:
I guess we can explicitly say `margin-top: 10rem;` in this case

Reply:
addressed with 36763544bb3d5db9544dc83e263cc92dbc78c739